deepchem/models/keras_model.py

```diff
@@ -186,28 +188,41 @@ def __init__(self,
       self.optimizer = optimizer
     self.tensorboard = tensorboard
 
+    self.loggers = logger
+    # Create a list of loggers
+    if self.loggers is not None:
+      if not isinstance(self.loggers, list):
+        # if not a list of loggers, make it a list of one logger
+        self.loggers = [self.loggers]
+    else:
+      self.loggers = []
+
     # W&B flag support (DEPRECATED)
     if wandb:
-      logger.warning(
-          "`wandb` argument is deprecated. Please use `wandb_logger` instead. "
-          "This argument will be removed in a future release of DeepChem.")
+      logs.warning(
+          "`wandb` argument is deprecated. Please use `logger` instead. "
+          "`wandb` will be removed in a future release of DeepChem.")
     if wandb and not _has_wandb:
-      logger.warning(
+      logs.warning(
           "You set wandb to True but W&B is not installed. To use wandb logging, "
-          "run `pip install wandb; wandb login`")
+          "run `pip install wandb` then log in using `wandb login` "
+          "or `import wandb; wandb.login()` in script.")
     self.wandb = wandb and _has_wandb
 
+    # Wandb_logger flag support (DEPRECATED)
```

Comment:
`wandb_logger` should be completely removed, replaced by `logger`.  It hasn't existed in any release, so we don't need to worry about maintaining backward compatibility.

Reply:
Sure, I can remove it. This is concerning existing in major releases right? As far as I know, it's currently still possible for people to use `wandb_logger` if they install deepchem using `pip install --pre deepchem`

Reply:
All the more reason to remove it now so we don't get stuck with another deprecated argument we have to support in the future.

Reply:
sure, I've removed it